nextjs-app/components/pages/main/Partners.tsx

```diff
@@ -1,21 +1,85 @@
 import Image from "next/image";
+import { FABLE_PARTNER_URL } from "@/constants/partners";
+
+const programPartners = [
+  {
+    name: "Fable",
+    imageSrc: "/images/partner-fable.png",
+    href: FABLE_PARTNER_URL,
+  },
+  {
+    name: "Fable",
+    imageSrc: "/images/partner-fable.png",
+    href: FABLE_PARTNER_URL,
+  },
+  {
+    name: "Fable",
+    imageSrc: "/images/partner-fable.png",
+    href: FABLE_PARTNER_URL,
+  },
+  {
+    name: "Fable",
+    imageSrc: "/images/partner-fable.png",
+    href: FABLE_PARTNER_URL,
+  },
+  {
+    name: "Fable",
+    imageSrc: "/images/partner-fable.png",
+    href: FABLE_PARTNER_URL,
+  },
+  {
+    name: "Fable",
+    imageSrc: "/images/partner-fable.png",
+    href: FABLE_PARTNER_URL,
+  },
+  {
+    name: "Fable",
+    imageSrc: "/images/partner-fable.png",
+    href: FABLE_PARTNER_URL,
+  },
+  {
+    name: "Fable",
+    imageSrc: "/images/partner-fable.png",
+    href: FABLE_PARTNER_URL,
+  },
+];
 
 const Partners = () => {
   return (
-    <section className="py-16 bg-[#F4F4F4]">
+    <section className="py-[72px] bg-neutral-1 md:py-[120px]">
       <div className="mx-auto text-center">
-        <h2 className="text-h4 text-[#7E7059]">合作夥伴</h2>
-        <h3 className="text-[48px] leading-11 font-semibold font-eb-garamond mb-6 text-[#090E3E]">
+        <h2 className="text-h5 md:text-h4 text-yellow-6 mb-2">合作夥伴</h2>
+        <h3 className="text-h3-title md:text-h1-title font-eb-garamond mb-6 md:mb-8 text-blue-8">
           Our Partners
         </h3>
         <Image
           src="/images/title-symbol-line.png"
           alt="title-symbol-line"
           width={57}
           height={5}
-          className="mx-auto mb-9"
+          className="mx-auto mb-11"
         />
-        <div className="flex flex-wrap justify-center gap-5"></div>
+        <div className="w-full px-5 md:px-10 xl:px-[142px]">
+          <div className="flex flex-wrap gap-4 md:gap-7 justify-center">
+            {programPartners.map((partner, idx) => (
+              <a
+                key={idx}
+                href={partner.href}
+                target="_blank"
+                rel="noopener noreferrer"
+                className="bg-white rounded-3 w-[calc(50%-6px)] md:w-[calc(50%-12px)] lg:w-[calc(25%-18px)]"
+              >
+                <Image
+                  src={partner.imageSrc}
+                  alt={`${partner.name} logo`}
+                  width={324}
+                  height={108}
+                  className="object-contain h-[108px] w-full"
+                />
+              </a>
+            ))}
+          </div>
```

Comment:
NIT: 

I think `grid` is suitable using here; that way, we don’t need to calculate the widths manually 😆


```suggestion
          <div className="grid grid-cols-2 lg:grid-cols-4 gap-4 md:gap-7">
            {programPartners.map((partner, idx) => (
              <a
                key={idx}
                href={partner.href}
                target="_blank"
                rel="noopener noreferrer"
                className="bg-white rounded-3"
              >
                <Image
                  src={partner.imageSrc}
                  alt={`${partner.name} logo`}
                  width={324}
                  height={108}
                  className="object-contain h-[108px] w-full"
                />
              </a>
            ))}
```

Reply:
Great idea ! This is my first time to use css grid ! Fixed